broker.yaml

```diff
@@ -0,0 +1,728 @@
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Error'
+    Forbidden:
+      description: 'Forbidden ([RFC 7231](https://tools.ietf.org/html/rfc7231))'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Error'
+    NotFound:
+      description: 'Not found ([RFC 7231](https://tools.ietf.org/html/rfc7231))'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Error'
+    InternalServerError:
+      description: 'Internal server error ([RFC 7231](https://tools.ietf.org/html/rfc7231))'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Error'
+    Error:
+      description: 'Unexpected error ([RFC 7231](https://tools.ietf.org/html/rfc7231))'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Error'
+  
+  schemas:
+    Service:
+      type: object
+      properties:
+        projectId:
+            type: string
+            description: Unique id generated for each project
+            example: project_12
+        callbackUrl:
+            type: string
+            description: URL on which CD is listening for image updates.
+            example: https://ec2-54-203-145-132.compute-1.amazonaws.com/update
+    BuildInfo:
+      type: object
+      properties:
+        commitSHA:
+          type: string
+          description: Last commit sha till where repo needs to be checkout by git
+          example: '6b9899f'
+        imageTag:
+          type: string
+          description: Tag name for the created image
+          example: 'v1'
+        branch:
+          type: string
+          description: Git branch to checkout
+          example: feature-xyz
+    ProjectListItem:
+      type: object
+      properties:
+          projectId:
+              type: string
+              description: Unique id generated for each project
+              example: project_12
+      allOf:
+        - $ref: '#/components/schemas/Project'
+    Project:
+      type: object
+      properties:
+        type:
+          type: string
+          description: Development type
+          example: production
+        url:
+          type: string
+          description: URL to the code repository
+          example: https://github.com/elixir-cloud-aai/trs-filer.git
+        dockerfileLocation:
+          type: string
+          description: Location of Dockerfile relative to repo root dir
+          example: docker/Dockerfile
+        containerRegistryUrl:
+          type: string
+          description: Location of container registry to push build image
+          example: gcr.io/my-project/busybox
+        containerRegistryToken:
+          type: string
+          description: Token to access container registry
+          example: XXXXXXXXXXXXXXX
+    Error:
```

Comment:
Please have a look and try to make this compliant with https://datatracker.ietf.org/doc/html/draft-ietf-appsawg-http-problem-00 in case it is not. Also, ideally incorporate any recommendations (`SHOULD`) from that spec, as well as best practices (e.g., you could include the `type` and `instance` properties as described).